docs/guides/setup/installing/local.md

```diff
@@ -1,5 +1,89 @@
 ---
-title: local python
+title: Installing in great_expectations in Local Python3
 ---
+import Prerequisites from '/docs/guides/setup/components/install_prereq.jsx'
+import NextSteps from '/docs/guides/setup/components/install_nextsteps.md'
+import Congratulations from '/docs/guides/setup/components/install_congrats.md'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-This article is a stub.
+This guide will help you Install Great Expectations locally for use with python
+
+<Prerequisites>
+
+</Prerequisites>
+
+## Steps
+
+### 1. Install required dependencies
+
+First, check that you have a python3 with pip installed
+
+```console
+python --version
+# or if multiple versions of python installed
+python3 --version
+python3 -m pip --version
+```
+
+### 2. Install Great Expectations
+```console
+pip install great_expectations
```

Comment:
Regarding installation, running `great_expectatsions --v3-api init` seems out of scope (and would be handled by a different doc). 

would we be able to have Step 2 be the following 

```
pip install great_expectations

great_expectations --version
```

And then have a link to 

`docs/guides/setup/configuring-data-contexts/how-to-create-a-new-data-context-with-the-cli.md?`

- this guide should cover the init and walking through the CLI steps.

Reply:
Done